### PR TITLE
Creates a page on SNI to fix #46.

### LINKED ISSUE #46: Server Name Indication
A page on Server Name Indication -- its benefits, its client support, and how to evaluate the desirability and feasibility of requiring it on one's domain.

### PR DESCRIPTION
**Pull request by willchan**
Initial pass on documenting SNI.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -33,6 +33,9 @@ navigation:
 - text: FAQ
   url: faq/
   internal: true
+- text: Server Name Indication
```
Comment (alex) [inline _config.yml]: Should we put `SNI` in parens? I think the initialism is at least as well known as the name. (The text may be too long with that, dunno)

**Comment by alex** [comment]
In general the text looks ok, but it feels a bit too technical / too many assumptions. I'm not sure there's any specific changes I can suggest though :-/ I'll try another read in the morning to see if I can be more concrete.

**Comment by konklone** [comment]
Great first pass! @willchan, would you mind doing this as an in-repo, branch-to-branch PR, instead of from a fork?

Or at least, just make all the Wikipedia links `https://`, we'll merge, and do branch-to-branch for future PRs. :smiley_cat:

**Comment by konklone** [comment]
To be more clear, the branch-to-branch PR would let us make edits in-PR before merge. And yeah, I think it could use some additional hand-holding for the concepts, but it's a great core to work from.

**Comment by willchan** [comment]
I was going to say that you have to invite me to your super cool organization first, but I see now that Gray left me an invite back in February. Oops. I'll kill this PR and recreate it as an in-repo PR tomorrow. Or someone else can do it (I don't mind if anyone wants to copy/paste the text and start from there). I'm going to bed now though :)

**Pull request closed without merging**